.github/CODEOWNERS

```diff
@@ -1279,6 +1279,7 @@ sdk/oep/arm-oep @qiaozha @dw511214992 @MaryGao
 /rush.json @mikeharder @praveenkuttappan
 /tsconfig.json @mikeharder @praveenkuttappan @witemple-msft @deyaaeldeen
 /**/tsconfig.json @witemple-msft @deyaaeldeen
+/**/tsconfig.strict.json @deyaaeldeen
```

Comment:
Are there other examples of `tsconfig.strict.json` in the repo?

Reply:
not the I know of, I believe `cosmosdb` is the last package that has this flag off.